src/dart-vm/index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Dart VM
-description: The landing page for all things relating to the Dart virtual machine (Dart VM).
+title: Server-Side Dart
+description: All things relating to command-line and server-side apps.
 toc: false
 ---
 
 This page points to tools and documentation
-that can help you develop scripts and server apps
-for the standalone Dart VM.
+that can help you develop command-line and server-side apps
+for the Dart VM.
```

Comment:
I'd drop "for the Dart VM" (the section title already makes it clear that we're focused on Dart apps).